tinygrad/device.py

```diff
@@ -184,8 +184,9 @@ def compile_cached(self, src:str) -> bytes:
     return lib
 
 class Compiled:
-  def __init__(self, device:str, allocator:Allocator, renderer:Optional[Renderer], compiler:Optional[Compiler], runtime, graph=None):
-    self.dname, self.allocator, self.compiler, self.runtime, self.graph = device, allocator, compiler or Compiler(), runtime, graph
+  def __init__(self, device:str, allocator:Allocator, renderer:Optional[Renderer],
+               compiler:Optional[Compiler], runtime, graph=None, count=0):
+    self.dname, self.allocator, self.compiler, self.runtime, self.graph, self.count = device, allocator, compiler or Compiler(), runtime, graph, count
```

Comment:
How is count not always 0?

Reply:
I added it so mypy wouldn't give the error on Device not having the count, but I agree it's not the cleanest of solutions. I'll close this PR now and come up with something better